clients/ts/signalr/tests/HttpConnection.test.ts

```diff
@@ -644,16 +644,16 @@ describe("HttpConnection", () => {
             let eventSourceConstructorCalled: boolean = false;
 
             const customEventSourceType = class EventSource {
-                constructor(url: string, eventSourceInitDict?: EventSourceInit) {
+                constructor() {
                     eventSourceConstructorCalled = true;
                     throw new Error("EventSource constructor called.");
                 }
             };
 
             const options: IHttpConnectionOptions = {
                 ...commonOptions,
-                EventSource: customEventSourceType as EventSourceConstructor,
-                httpClient: new TestHttpClient().on("POST", (r) => {
+                EventSource: customEventSourceType as any,
```

Comment:
TS compiler was failing because our custom EventSource type defined in this test doesn't actually implement all the things in EventSource. I guess casting doesn't really satisfy it, but casting to `any` is basically like telling the TS compiler to leave us alone.